src/backend/egl/display.rs

```diff
@@ -694,10 +698,16 @@ unsafe impl Send for EGLBufferReader {}
 
 #[cfg(feature = "use_system_lib")]
 impl EGLBufferReader {
-    fn new(display: Arc<EGLDisplayHandle>, wayland: *mut wl_display) -> Self {
+    fn new<L>(display: Arc<EGLDisplayHandle>, wayland: *mut wl_display, log: L) -> Self
+    where
+        L: Into<Option<::slog::Logger>>,
+    {
+        let logger = crate::slog_or_fallback(log.into()).new(o!("smithay_module" => "backend_egl"));
```

Comment:
Could you make the module as `"egl_buffer_reader"` instead, to be more specific?